packages/react-components/src/AccountIndex.tsx

```diff
@@ -32,7 +32,7 @@ function AccountIndex ({ children, className = '', defaultValue, label, value }:
     [info]
   );
 
-  if (!api.query.indices) {
+  if (!api || !api.query.indices) {
```

Comment:
can we replace `!api || !api.query.indices` with `!api?.query.indices`

Reply:
done